src/plugins/widgets/workHours/WorkHours.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from "react";
+import { useTime } from "../../../hooks";
+import { Props, defaultData } from "./types";
+
+const WorkHours: FC<Props> = ({data = defaultData}) => {
+  const start = buildDateTime(data.startTime);
+  const end = buildDateTime(data.endTime);
+  const time = useTime();
+
+ return (
+   <div className="WorkHours">
+     <h2>{hoursProgress(time, start, end)}</h2>
+   </div>
+ )
+}
+
+const hoursProgress = (current: Date, start: Date, end: Date): string => {
```

Comment:
```suggestion
const hoursProgress = (current: Date, start: Date, end: Date): number => {
```
You could return a number here, and add the `%` sign in the main component instead for a cleaner function.